tests/checks/tmux-empty-prompt.fish

```diff
@@ -0,0 +1,77 @@
+#RUN: %fish %s
+#REQUIRES: command -v tmux
+
+isolated-tmux-start -C '
+    function fish_prompt; end
+    function fish_right_prompt
+        set -q right_prompt
+        or echo right-prompt
+    end
+    bind ctrl-g "set -q right_prompt; and set -e right_prompt; or set right_prompt 1; commandline -f repaint"
```

Comment:
I'd probably write this in multiple lines

	bind ctrl-g "
		set -q right_prompt
		and set -e right_prompt
		or set right_prompt 1
		commandline -f repaint
	"